src/app/dashboard/index.component.ts

```diff
@@ -39,67 +43,25 @@ import { TasksStatusesGroup } from './types';
   <mat-icon matListItemIcon aria-hidden="true" [fontIcon]="getPageIcon('dashboard')"></mat-icon>
   <span i18n="Page title"> Dashboard </span>
 </app-page-header>
-
-<app-page-section>
-  <app-page-section-header icon="adjust">
-    <span i18n="Section title"> Tasks by status </span>
-  </app-page-section-header>
-
-  <mat-toolbar>
-    <app-actions-toolbar>
-      <app-actions-toolbar-group>
-        <app-refresh-button [tooltip]="autoRefreshTooltip()" (refreshChange)="onRefresh()"></app-refresh-button>
-        <app-spinner *ngIf="loadTasksStatus"></app-spinner>
-      </app-actions-toolbar-group>
-
-      <app-actions-toolbar-group>
-        <app-auto-refresh-button [intervalValue]="intervalValue" (intervalValueChange)="onIntervalValueChange($event)"></app-auto-refresh-button>
-
-        <button mat-icon-button [matMenuTriggerFor]="menu" aria-label="Show more options">
-          <mat-icon aria-hidden="true" [fontIcon]="getIcon('more')"></mat-icon>
-        </button>
-        <mat-menu #menu="matMenu">
-          <button mat-menu-item (click)="onToggleGroupsHeader()">
-            <mat-icon aria-hidden="true" [fontIcon]="hideGroupHeaders ? getIcon('view') : getIcon('view-off')"></mat-icon>
-            <span i18n>
-              Toggle Groups Header
-            </span>
-          </button>
-          <button mat-menu-item (click)="onManageGroupsDialog()">
-            <mat-icon aria-hidden="true" [fontIcon]="getIcon('tune')"></mat-icon>
-            <span i18n>
-              Manage Groups
-            </span>
-          </button>
-        </mat-menu>
-      </app-actions-toolbar-group>
-    </app-actions-toolbar>
-  </mat-toolbar>
-
-  <div class="groups">
-    <app-statuses-group-card
-      *ngFor="let group of statusGroups"
-      [group]="group"
-      [data]="data"
-      [hideGroupHeaders]="hideGroupHeaders"
-    ></app-statuses-group-card>
-  </div>
-</app-page-section>
+<section class="lines" *ngFor="let line of lines" >
+  <app-line 
+      [line]="line" 
+      (saveChange)="onSaveChange()"
+      (filtersChange)="onSaveChange()"
+      (toggleGroupsHeaderChange)="onSaveChange()"
+      (manageGroupsDialogChange)="onSaveChange()"
+  ></app-line>
+</section>
   `,
   styles: [
     `
-app-actions-toolbar {
-  display: block;
-  width: 100%;
-}
-
-.groups {
-  margin-top: 1rem;
-
-  display: grid;
-  grid-template-columns: repeat(3, 1fr);
-  grid-gap: 1rem;
-}
+      app-line {
+        margin: 1em; 
+      }
```

Comment:
You could remove space before to stick content to the left side.
```suggestion
      }
      
```